.github/workflows/e2e-leg-1.yml

```
@@ -79,5 +86,5 @@ jobs:
     - uses: actions/upload-artifact@v3
       if: failure()
       with:
-        name: logs-e2e-leg-1-${{ inputs.vertica-deployment-method }}
+        name: logs-e2e-leg-1-${{ inputs.vertica-deployment-method }}-${{ inputs.artifact-version }}
```

If we don't specify this new parm, the artifact name seems a little odd (e.g. logs-e2e-leg-1-admintools-). Could we call it artifact-suffix then just add it on without a dash? This would allow us to omit this parameter and still come up with a good name.
```suggestion
        name: logs-e2e-leg-1-${{ inputs.vertica-deployment-method }}${{ inputs.artifact-suffix }}
```

---

Yes, It will be better, You're right, the artifact name seems a little odd if we don't specify the new param.